components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-overview.tsx

```diff
@@ -182,20 +236,26 @@ export const PortfolioOverview = () => {
       )
   }, [
     filteredOutPortfolioNetworkKeys,
-    userVisibleTokensInfo
+    userTokensWithRewards
   ])
 
   const userVisibleNfts = React.useMemo(() => {
     return visibleTokensForFilteredChains.filter(token => token.isErc721 || token.isNft)
   }, [visibleTokensForFilteredChains])
 
+  const networksList =
```

Comment:
memoize arrays to prevent child rerenders

Reply:
Memoized